sdk/communication/Azure.Communication.Rooms/tests/RoomsClient/RoomsClientLiveTests.cs

```diff
@@ -153,6 +153,30 @@ public async Task AcsRoomLifeCycleLiveTest()
         }
 
         [Test]
+        public async Task GetRoomsLiveTest()
```

Comment:
Hey Minnie, for the name of the test. I would suggest using the following standards. I hope we can do the same for other languagues too. This would make it easier to maintenance across the boards. 

With this test method, we can name it something like this GetRoomsLiveTest_FirstRoomIsNotNull_Succeed

The name of the method being tested.
The scenario under which it's being tested.
The expected behavior when the scenario is invoked

Reply:
Done, great suggestion